hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/legacy/core/jproto/JKey.java

```diff
@@ -312,7 +312,7 @@ public static Key convertJKey(JKey jkey, int depth) throws InvalidKeyException {
         if (depth > MAX_KEY_DEPTH) {
             throw new InvalidKeyException("Exceeding max expansion depth of " + MAX_KEY_DEPTH);
         }
-        if (!(jkey.hasThresholdKey() || jkey.hasKeyList() || jkey.isEmpty())) {
```

Comment:
This triggered an infinite recursion without isEmpty.
Do we need special handling for empty JKeys?